src/services/ui/src/pages/detail/index.tsx

```diff
@@ -39,19 +39,31 @@ const DetailCardWrapper = ({
 const StatusCard = ({
   status,
   raiWithdrawEnabled,
+  raiRecievedDate,
 }: {
   status: string;
   raiWithdrawEnabled: boolean;
-}) => (
-  <DetailCardWrapper title={"Status"}>
-    <div>
-      <h2 className="text-xl font-semibold">{status}</h2>
-      {raiWithdrawEnabled && (
-        <em className="text-xs">{"Withdraw Formal RAI Response - Enabled"}</em>
-      )}
-    </div>
-  </DetailCardWrapper>
-);
+  raiRecievedDate: string;
+}) => {
+  const transformedStatuses = getStatus(SEATOOL_STATUS.WITHDRAWN);
+  return (
+    <DetailCardWrapper title={"Status"}>
+      <div>
+        <h2 className="text-xl font-semibold">{status}</h2>
+        {raiWithdrawEnabled &&
+        !Object.values(transformedStatuses).includes(status) ? (
+          <em className="text-xs">
+            {"Withdraw Formal RAI Response - Enabled"}
+          </em>
+        ) : null}
+        {/* Display 2nd Clock if status is pending and latestRaiResponseTimestamp is present */}
+        {status.toLowerCase() === "pending" && raiRecievedDate && (
```

Comment:
This logic is faulty. We're expecting "any pending status except Pending - RAI", and this is saying "exclusively the 'pending' status".

You'll also have to ensure this is not shown for CHIP SPAs per the AC. I didn't see any checks on package type.